spec/importc.dd

```diff
@@ -0,0 +1,14 @@
+Ddoc
+
+$(SPEC_S ImportC,
+
+$(HEADERNAV_TOC)
+
+$(P Documentation for ImportC coming soon!)
+
+$(SPEC_SUBNAV_PREV_NEXT betterC, Better C, ob, Live Functions)
```

Comment:
This results in a broken link on case sensitive filesystems, I think the correct line would be:
```suggestion
$(SPEC_SUBNAV_PREV_NEXT betterc, Better C, ob, Live Functions)
```

Reply:
that's why I'm doing this boilerplate stuff as a separate PR! Done.